src/System.Runtime/tests/System/GCTests.cs

```diff
@@ -399,6 +404,7 @@ public static void LargeObjectHeapCompactionModeRoundTrips(GCLargeObjectHeapComp
         [Theory]
         [InlineData(GCLatencyMode.Batch)]
         [InlineData(GCLatencyMode.Interactive)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Mono)]
```

Comment:
GCLatencyMode is just a performance hint. It should be trivial to make this property work as no-op on Mono.